src/System.Private.CoreLib/shared/System/SpanHelpers.T.cs

```diff
@@ -53,6 +50,63 @@ public static int IndexOf<T>(ref T searchSpace, int searchSpaceLength, ref T val
             return -1;
         }
 
+        // Adapted from IndexOf(...)
+        public static unsafe bool Contains<T>(ref T searchSpace, T value, int length)
```

Comment:
I don't think this method needs to be marked as unsafe.

Reply:
We missed this in the CR. Will submit a separate PR.